ToDoList-frontend/src/app/bucket/bucket.component.css

```diff
@@ -25,4 +25,11 @@
   select {
     width: 100%;
     padding: .5rem;
+  }
+
+  .search-icon {
```

Comment:
:/ Uzyjmy tego https://github.com/FortAwesome/angular-fontawesome

Reply:
no ale to musi tak być osadzone w tym stylu żeby wyrównać ikonkę lupki do prawej, a ma jakieś znaczenie czy heroicon czy fortawesome?

Reply:
Chodzilo mi glownie o ten background image i dlugi "URL". Jesli sie uzyje tej paczki, to nie trzeba bedzie w ogole pisac takich dlugich "URLi".

Reply:
Niestety użycie biblioteki font-awesome nie za wiele tutaj pomoże. Input musi być stylizowany sprytnie css'em i nawet jakbyśmy chcieli zamienić tego url w taki sposób:
  .search-icon {
    background-image: '<fa-icon [icon] = "faSearch"></fa-icon>';
    background-repeat: no-repeat;
    background-position: right;
    padding-right: 25px;
  }
 to ikonka w odpowiednim miejscu nie będzie wyświetlona. Odpowiedź z chata GPT:
 _The background-image property expects a valid URL or an image reference, not HTML markup. The value you provided, <fa-icon [icon]="faSearch"></fa-icon>, is Angular markup and cannot be used directly as a background image._